client/src/components/ProductDetails/ProductOverview.jsx

```diff
@@ -22,24 +26,41 @@ function ProductOverview ({currentProduct, productStylesArray, currentStyleID, c
           <StyleSelector
             productStylesArray={productStylesArray}
             currentStyleID={currentStyleID}
-            changeCurrentStyle={changeCurrentStyle}/>
+            changeCurrentStyle={changeCurrentStyle}
+            currentProduct={currentProduct}/>
           <DropDownsAndButtons
             productStylesArray={productStylesArray}
             currentStyleID={currentStyleID}/>
         </div>
+
       </div>
 
-      <div className="d-flex flex-row" style={{width: '50em', margin: '1em', position: 'relative'}}>
-        <div>
+
+      <div className="d-flex flex-row">
+
+        <div className="flex-column" style={{width: '50em', margin: '1em', position: 'relative'}}>
```

Comment:
Do you want to add a className to this div and move the corresponding inline styling to your CSS file?

Reply:
yes, ill give this a go. Lets also chat about consolidating all CSS files into one. Should be a quick refactor